### PR TITLE
The sign in form is created

### PR DESCRIPTION
**Pull request by dmitryshunto**

### CODE REVIEW

**Comment by eugenmolchanov** [comment]
Task linked: [CU-861me3nvm Create login form](https://app.clickup.com/t/861me3nvm)

File: src/Components/Navbar/NavbarItem/NavbarItem.tsx [hunk 1]
```diff
@@ -9,7 +9,8 @@ export type NavbarItemType = {
   
 const NavbarItem: React.FC<NavbarItemType> = (props) => {
     return (
-        <Link 
+        <Link
+            data-testid={props.route}
```
Comment (eugenmolchanov) [inline src/Components/Navbar/NavbarItem/NavbarItem.tsx:13]: Why do we use `testid` here?

**Review by eugenmolchanov — COMMENTED** [review]
SonarCloud reports 20% coverage but it includes test classes in the report as uncovered. They don't have to be included. May you fix that in this PR?
Also SonarCloud reported one code smell. The general rule of thumb is to fix any code smells and bugs reported by SonarCloud in the scope of the same PR.
And could you please attach the screenshots of the pages in the PR description to see for the reviewers how they are going to look like?

File: src/Components/Navbar/NavbarItem/NavbarItem.tsx (hunk 1 above)
Comment (dmitryshunto) [reply src/Components/Navbar/NavbarItem/NavbarItem.tsx:13]: We use it for element selecting in App.test.tsx. I had some problems with this test and I tried differnt ways of element selecting.

**Review by eugenmolchanov — APPROVED** [review]

**Pull request merged**